Makefile

```diff
@@ -166,25 +140,10 @@ firmwares: stamp-clean-firmwares .stamp-firmwares
 	  FEED_REVISION=`cd $$FEED_DIR; $(REVISION)`; \
 	  echo "Feed $$FEED: repository from $$FEED_GIT_REPO, git branch \"$$FEED_GIT_BRANCH_ESC\", revision $$FEED_REVISION" >> $$VERSION_FILE; \
 	done
-	# copy different firmwares (like vpn, minimal) including imagebuilder
-	for DIR_ABS in $(IB_BUILD_DIR)/imgbldr/bin/*; do \
-	  TARGET_DIR=$(FW_TARGET_DIR)/$$(basename $$DIR_ABS); \
-	  rm -rf $$TARGET_DIR; \
-	  mv $$DIR_ABS $$TARGET_DIR; \
-	  cp $(FW_TARGET_DIR)/$$VERSION_FILE $$TARGET_DIR/; \
-	  for FILE in $$TARGET_DIR/openwrt*; do \
-	    [ -e "$$FILE" ] || continue; \
-	    NEWNAME="$${FILE/openwrt-/kathleen-}"; \
-	    NEWNAME="$${NEWNAME/ar71xx-generic-/}"; \
-	    NEWNAME="$${NEWNAME/mpc85xx-generic-/}"; \
-	    NEWNAME="$${NEWNAME/squashfs-/}"; \
-	    mv "$$FILE" "$$NEWNAME"; \
-	  done; \
-	done;
+	./assemble_firmware.sh -p "$(PROFILES)" -i $(IB_FILE) -t $(FW_TARGET_DIR) -u "$(PACKAGES_LIST_DEFAULT)"
```

Comment:
@lynxis I take it you haven't had a look at https://github.com/freifunk-berlin/firmware/commit/fa4aafa7e594833efd18d113ab24dda77cfe9918 and https://github.com/freifunk-berlin/firmware/issues/178 . `git blame` makes finding that really easy.

Aside from some routers having problems with filenames that long, we're not building _openwrt_ (or LEDE) images here, we're building _kathleen_ images. Also, look at the mess that http://buildbot.berlin.freifunk.net/buildbot/unstable/ar71xx-generic/2/default/ is currently. Speaking of that, http://buildbot.berlin.freifunk.net/buildbot/unstable/ is quite messy now, too. Sigh. I'd love if we could at some point work on the firmware, not on its build infrastructure.